jupyterlab/src/index.ts

```diff
@@ -4,9 +4,7 @@ import {
 } from "@jupyterlab/application";
 
 import { IEditorLanguageRegistry } from "@jupyterlab/codemirror";
-// @ts-expect-error No typings avaiable for @codemirror/legacy-modes
 import { simpleMode } from "@codemirror/legacy-modes/mode/simple-mode";
-// @ts-expect-error No typings avaiable for @codemirror/language
 import { LanguageSupport, StreamLanguage } from "@codemirror/language";
```

Comment:
New settings made these errors go away. Typings _do_ exist for these modules, maybe TypeScript was failing to discover them before?